src/pytorch_ie/pipeline.py

```diff
@@ -331,6 +331,9 @@ def __call__(
         # This creates encodings from the documents. It modifies the documents and may produce multiple entries per
         # document.
         model_inputs = self.preprocess(documents, **preprocess_params)
+        if forward_params.pop("fast_dev_run", False):
+            warnings.warn("Execute a fast dev run, only the first model input will be processed.")
+            model_inputs = model_inputs[:1]
```

Comment:
I think we should use two model inputs, if available. This will uncover issues resulting from batching. I think pytorch lightning does that too.

Reply:
fixed in 66b3b81. @ChristophAlt could you have another look?